website/static/js/osfLanguage.js

```diff
@@ -80,5 +80,10 @@ module.exports = {
         dataSendError: 'Error sending data to the server: check that all fields are valid, or contact ' + SUPPORT_LINK + ' if the problem persists.',
         creationSuccess: 'Successfully registered new application',
         dataUpdated: 'Application data updated'
+    },
+    projectSettings: {
+        updateSuccessMessage: 'Category, title, and description updated successfully',
+        updateErrorMessage: 'Error updating project settings',
```

Comment:
@abought What do you think? 
Did we talk about an alternative to asking to the user to contact support? 
Or should we keep it like this? 
If it's not a 500, should the message be different (more specific)?